tests/src/Common/external/external.depproj

```diff
@@ -21,21 +21,11 @@
     <PackageReference Include="Microsoft.CodeAnalysis.Compilers">
       <Version>1.1.1</Version>
     </PackageReference>
-    <PackageReference Include="xunit.performance.api">
-      <Version>1.0.0-beta-build0003</Version>
-    </PackageReference>
-    <PackageReference Include="xunit.performance.core">
-      <Version>1.0.0-beta-build0003</Version>
-    </PackageReference>
-    <PackageReference Include="xunit.performance.execution">
-      <Version>1.0.0-beta-build0003</Version>
-    </PackageReference>
-    <PackageReference Include="xunit.performance.metrics">
-      <Version>1.0.0-beta-build0003</Version>
-    </PackageReference>
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent">
-      <Version>1.0.3-alpha-experimental</Version>
-    </PackageReference>
+    <PackageReference Include="xunit.performance.api" Version="1.0.0-beta-build0004" />
```

Comment:
nit:  Maybe we should match how the rest of the package references are made.  I don't care if they are all single line style or the old multi-line style.